modP2pImpl/src/org/aion/p2p/impl1/tasks/TaskSend.java

```diff
@@ -67,17 +63,14 @@ public TaskSend(
         this.start = _start;
         this.nodeMgr = _nodeMgr;
         this.selector = _selector;
-
-        if (tpe == null) {
-            tpe =
-                    new ThreadPoolExecutor(
-                            TOTAL_LANE,
-                            TOTAL_LANE,
-                            0,
-                            TimeUnit.MILLISECONDS,
-                            new LinkedBlockingQueue<>(THREAD_Q_LIMIT),
-                            Executors.defaultThreadFactory());
-        }
+        this.tpe =
+                new ThreadPoolExecutor(
+                        1,
+                        1,
```

Comment:
will this change impact performance?

Reply:
I'm also curious about the performance impact.  Actually, why do we need to change this from TOTAL_LANE to 1?

Reply:
Before bug fix, the Thread executor was shared with whole Tasksend thread. But in this PR, each Tasksend thread has one dedicated thread to process the write message. (please see the runnable submit in the thread)